include/mrdocs/Metadata/Source.hpp

```diff
@@ -36,6 +36,15 @@ MRDOCS_DECL
 std::string_view
 toString(FileKind kind);
 
+namespace dom {
+    template <>
+    struct ToValue<FileKind>
```

Comment:
why specialization instead of tag_invoke?

Reply:
done :)